bundles/org.openhab.binding.hdpowerview/README.md

```diff
@@ -88,6 +98,13 @@ All of these channels appear in the binding, but only those which have a physica
 | batteryVoltage | Number:ElectricPotential | Battery voltage reported by the shade. |
 | signalStrength | Number                   | Signal strength (0 for no or unknown signal, 1 for weak, 2 for average, 3 for good or 4 for excellent) |
 
+### Channels for Repeaters (Thing type `repeater`)
+
+| Channel         | Item Type | Description                   |
+|-----------------|-----------|-------------------------------|
+| identify        | Switch    | Flash repeater to identify. Setting this to ON will identify the repeater by blinking. Note: include `{autoupdate="false"}` in the item configuration to avoid having to reset it to off after use. |
```

Comment:
You could avoid the autoupdate quirk by making this a command channel. See https://www.openhab.org/docs/developer/bindings/thing-xml.html#command-description

Reply:
Thanks for this hint. I agree this is more elegant. Changed and tested. I created only one command, **IDENTIFY**, and set the state back to **UNDEF** after three seconds, which seems to match the period where the repeater blinks. I considered **NONE** as command, but IMHO it doesn't make sense, since the blinking can't be stopped anyway. How does it look for you?